include/macros/me_script.inc

```diff
@@ -0,0 +1,88 @@
+	.macro me_nop
```

Comment:
I'd prefer the filename to be verbose: `mystery_event_script.inc`.  The macro names are fine abbreviated.